app/src/main/java/com/example/tmdb/modules/FavoritesViewModel.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.tmdb.modules
+
+import androidx.lifecycle.ViewModel
+
+class FavoritesViewModel(): ViewModel() {
+    val popularList = MovieRepositoryImpl(movieApi = MovieApiImpl()).getPopularMovies()
```

Comment:
Pass the `MovieRepository` via constructor to the view model. That way you won't need to create an instance here.